README.md

```diff
@@ -235,6 +235,23 @@ projector tutorial](https://www.tensorflow.org/tutorials/text/word_embeddings).
 The Text Dashboard displays text snippets saved via `tf.summary.text`. Markdown
 features including hyperlinks, lists, and tables are all supported.
 
+### Time Series Dashboard (new)
```

Comment:
I might omit `(new)` - it is new-*er* yes, but not *that* new (released in 2.4.0 last November), and this README is already so stale that I foresee this still saying `(new)` 10 years from now if left in ;)

Reply:
Done.